examples/example.php

```diff
@@ -64,7 +78,7 @@
 //    $line2->setVat('06');
 //    $sale->addLine($line2);
 //
-    $response = $teamleader->opportunitiesAddSale($sale);
+//    $response = $teamleader->opportunitiesAddSale($sale);
```

Comment:
What do you mean, should that line remain commented or not? I propably have commented it because you already have a $response object from ->helloWorld